cmd/openshift-tests/openshift-tests.go

```diff
@@ -109,7 +111,7 @@ func main() {
 }
 
 func newRunMonitorCommand() *cobra.Command {
```

Comment:
I think you renamed this to `monitor_command.NewRunMonitorCommand` so this code is probably no longer needed (it's definitely not called).

Same applies to `monitor_command.NewMonitorCommand` and `newMonitorCommand`.

Reply:
fixed